core-bundle/src/Cron/Cron.php

```diff
@@ -41,10 +41,42 @@ public function addCronJob(CronJob $cronjob): void
         $this->cronJobs[] = $cronjob;
     }
 
+    /**
+     * @return array<CronJob>
```

Comment:
Should we use `list<…>` from now on?

Reply:
Yeah, I wasn't sure (also since my IDE could not deal with `list<…>` 😁 )

Reply:
As it is non-standard I am not sure whether we should use it. And as I said, my IDE at least won't recognize it and thus it would not be able to know the type of the array elements (which would of course be inconvenient for me).

@contao/developers wdyt?